### PR TITLE
Introduce Prettier/ESLint, fix linter errors

### PR DESCRIPTION
**Pull request by martintosney**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,49 +1,40 @@
...
-    )
-  }
-}
+//     return (
+//       <button className={variant} {...others}>
+//         {content}
+//       </button>
+//     );
+//   }
+// }
```
Comment (burns238) [inline src/App.js:23]: :O Does it comment out any code it sees isn't used?

**Comment by burns238** [comment]
I take it you tested it? It all looks fine. Though I think we now know it was definitely me doing 4 space indentation - I blame python.

**Review by burns238 — APPROVED** [review]

File: src/App.js (hunk 1 above)
Comment (martintosney) [reply src/App.js:23]: > I take it you tested it? It all looks fine. Though I think we now know it was definitely me doing 4 space indentation - I blame python.

I'm pretty sure vim wanted me to use 4 spaces too, so you're not alone.

> :O Does it comment out any code it sees isn't used?

Not sure what happened there. I'll fix it before merging.

Comment (martintosney) [reply src/App.js:23]: Oh, I see what happened (this split confused me). Yeah, I manually commented this out to fix a linting error.

**Pull request merged**